package.json

```diff
@@ -40,7 +40,7 @@
         "husky": "^7.0.0",
         "jest": "^29.7.0",
         "lint-staged": "^12.1.2",
-        "prettier": "2.7.1",
+        "prettier": "2.8.1",
```

Comment:
so we can use `satisfies`

Reply:
test action is failing bc of some formatting issues after prettier upgrade - this PR: https://github.com/lightdash/lightdash/pull/12731 should be merged first